test/info.py

```diff
@@ -125,6 +126,7 @@
     (   0x7404,     'lpc11u35_6lowpan_borderrouterethernet_if', None,               'mbed-6LoWPAN-Border-Router-ETHERNET'   ),
     (   0xC000,     'lpc11u35_cocorico_if',                     None,               None                                    ), # TODO - set target to 'CoCo-ri-Co' when mbed-os supports this
     (   0x5500,     'lpc11u35_gr_peach_if',                     None,               'Renesas-GR-PEACH'                      ),
+    (   0x6660,     'lpc11u35_nz32_sc151_if',                   None,               'NZ32-SC151'                            ),
```

Comment:
I don't see ```NZ32-SC151``` supported in mbed-os. To run testing on the target mbed-os support is needed. To allow testing now just set target to ```None``` and leave a comment to the right indicating what the name would be.